src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue12642.cs

```diff
@@ -19,12 +18,11 @@ public Issue12642(TestDevice testDevice) : base(testDevice)
 		[Category(UITestCategories.Compatibility)]
 		public void ClickingQuicklyBetweenTopTabsBreaksContent()
 		{
+			App.WaitForElement("AutomatedRun");
```

Comment:
Using the `timeout` (TimeSpan) parameter, could help to wait during N seconds to the element and complete the Catalyst tab changing animation?

Reply:
@jsuarezruiz , Tried by adding timespan still getting OpenQA.Selenium.InvalidSelectorException on line no 25.

Reply:
Ok, could you try something like:

`App.QueryUntilPresent(() => App.WaitForElement("AutomatedRun"));`


QueryUntilPresent will retry the query until 10 times.

Reply:
@jsuarezruiz , I tried your suggestion, but unfortunately, I'm still encountering the same error. I found some code that functions correctly in Catalyst: App.WaitForElement(AppiumQuery.ById("Success")). Additionally, I was able to resolve some failing test cases in Catalyst by using AppiumQuery.ById. Could you please review this and let me know if I should commit these changes or if there’s anything else that needs to be fix generally?